Source/WebKit/Shared/WebUnifiedTextReplacementContextData.h

```diff
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <WebCore/AttributedString.h>
+#include <WebCore/CharacterRange.h>
```

Comment:
Do you need these includes? I think you can get away with forward decls and that would also satiate the GTK builders

Reply:
idk why GTK is even trying to find this file, considering everywhere that uses it is behind an ifdef